src/app/pages/questions/components/question/text-input/text-input.component.ts

```diff
@@ -111,11 +129,43 @@ export class TextInputComponent implements OnInit {
     return values.map(d => (d < 10 ? '0' + d : d)).map(String)
   }
 
+  datePickerObj: any = {}
+  selectedDate: string = new Date().toLocaleDateString('en-ZA')
```

Comment:
I used 'en-ZA' format  at first is to make sure when parsing data, **I can use `split` to get the data specifically**, but if we want to show the date format according to user's local date format, It might be a bit tricky. 
I updated the code, maybe there's a better way of doing this. @mpgxvii   Please help me take a look!

Reply:
Alternative: `new Date().toISOString().split('T')[0]` or `moment().format('YYYY-MM-DD')`